dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM java:8  
+
+COPY . /var/www/java  
+
+WORKDIR ./src
+
+ADD https://images.pexels.com/photos/4587959/pexels-photo-4587959.jpeg
```

Comment:
Please replace the hyperlink with a locally stored `jpeg` file to appease corporate demand for cloud scalability while avoiding cross-hosting headaches as well as, God-forbid, 404 missing file issues if pexels.com goes offline.

Here is the cat image, enhanced and upscaled using Machine Learning:
https://we.tl/t-okV1rNmg3d

You should add to the base of the project and then change line #7 of the Dockerfile as follows:
```
- ADD https://images.pexels.com/photos/4587959/pexels-photo-4587959.jpeg
+ ADD cat.jpeg
```

Reply:
Thank you very much for your work so we doing huge steps forward.

Another big problem here is the copyright of the cat image. We dont have a written contract to use this image in our software and we should avoid getting sued for 120m damages.

So i propose to replace the cat image with a ai generated cat image that has no copyright.

Reply:
I agree that a copyright free image would be the best option here.

However, we should be cognizant to the developing field of AI-rights wherein the ownership of AI-generated works of art may belong to the AI, or perhaps the creator of said AI, and not necessarily the public domain. 

We should acquire an AI-generated cat image and revisit this issue of legality in 20 years from today's date. Put this on your calendars.

Reply:
How about we use a cat image that I took? Copyright: me.

![88100EFF-0091-4076-84CB-6AD0E0F3E67C_1_105_c](https://user-images.githubusercontent.com/20162718/174499665-2377f3d1-25c8-4494-b69c-f431e4c18a65.jpeg)

Reply:
Can you prove the ownership and sign a contract that is added to the docker image?

Reply:
The screenshot below shows that I have similar images taken at the same time of the same cat (I also have dozens of other pictures of this cat [his name is Kimchi 🐱]).

![Screen Shot 2022-06-19 at 3 52 41 PM](https://user-images.githubusercontent.com/20162718/174503294-7f609a3c-e9bb-4e1c-82ae-bab4f2002aca.png)

I don't know if EXIF data counts as proof, but here's a screenshot of the cat image with its exif data:
![Screen Shot 2022-06-19 at 3 55 15 PM](https://user-images.githubusercontent.com/20162718/174503404-81bb746b-a830-4927-8308-434277bf2b40.png)

Reply:
I'm also happy to fly out to our Enterprise Headquarters in the corporate private jet to provide additional proof.

Reply:
I guess that is enough. We should use a picture of Kimchi.

Reply:
I believe relying on pexels is not sustainable; I'd commit the image into directly the repository instead.
But I do agree we should investigate why the Dockerfile is not working unless we add more cats to it.

Reply:
>But I do agree we should investigate why the Dockerfile is not working unless we add more cats to it.

We did the research a year ago. This project needs to cut some costs so we cant do it again. One image of kimchi would not add much bloat to the current state of our goal.